packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/components/idea/IdeaFrame.kt

```diff
@@ -0,0 +1,93 @@
+/**
+ * Represents the main frame of IDEA. The one you see when you open a project, the whole window.
+ * Reference code taken from:
+ * https://github.com/JetBrains/intellij-ui-test-robot/blob/master/ui-test-example/src/test/kotlin/org/intellij/examples/simple/plugin/pages/IdeaFrame.kt
+ */
+
+package com.mongodb.jbplugin.fixtures.components.idea
+
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.data.RemoteComponent
+import com.intellij.remoterobot.fixtures.*
+import com.mongodb.jbplugin.fixtures.findVisible
+
+/**
+ * Fixture that represents the frame itself. You can add more functions here if you want to interact
+ * with the frame.
+ *
+ * @param remoteRobot
+ * @param remoteComponent
+ */
+@FixtureName("Idea frame")
+@DefaultXpath("IdeFrameImpl type", "//div[@class='IdeFrameImpl']")
+class IdeaFrame(
+    remoteRobot: RemoteRobot,
+    remoteComponent: RemoteComponent,
+) : CommonContainerFixture(remoteRobot, remoteComponent) {
+    fun openFile(path: String) {
```

Comment:
I'm not sure where this is being used, do we need to worry about e.g. path traversal issues?

Reply:
It's only for tests, it's in the test folder and likely it's going to be always hardcoded. The worst thing that can happen is a failing test. It's not bundled anyhow with the plugin.